packages/flutter/lib/src/widgets/default_text_editing_shortcuts.dart

```diff
@@ -402,22 +402,16 @@ class DefaultTextEditingShortcuts extends StatelessWidget {
     SingleActivator(LogicalKeyboardKey.arrowLeft, alt: true): DoNothingAndStopPropagationTextIntent(),
     SingleActivator(LogicalKeyboardKey.arrowRight, alt: true): DoNothingAndStopPropagationTextIntent(),
     SingleActivator(LogicalKeyboardKey.arrowUp, alt: true): DoNothingAndStopPropagationTextIntent(),
-    SingleActivator(LogicalKeyboardKey.arrowDown, shift: true, alt: true): DoNothingAndStopPropagationTextIntent(),
```

Comment:
I believe you're correct, it does look like this would affect the `_macDisablingTextShortcuts` so we probably want to preserve behavior for `alt + shift + up/down`